docs/Config.md

```diff
@@ -181,9 +181,11 @@ gui:
   # If true (default), file icons are shown in the file views. Only relevant if NerdFontsVersion is not empty.
   showFileIcons: true
 
-  # Whether to show full author names or their shortened form in the commit graph.
-  # One of 'auto' (default) | 'full' | 'short'
-  # If 'auto', initials will be shown in small windows, and full names - in larger ones.
+  # How to show author names in the Commits view.
+  # One of 'auto' (default) | 'full' | 'short | 'truncateTo:<n,m>'
+  # If 'short', show initials both in the normal view and the expanded view.
+  # If 'auto', initials will be shown in the normal view, and full names (truncated to 17 characters) in the expanded view.
+  # If 'truncateTo:<n,m>', truncate to n characters in the normal view and m characters in the expanded view.
```

Comment:
Actually I also support `truncateTo:n`, in which case only the length for the normal view is configured, and the one for the expanded view is left at the default of 17. I find this useful, but I found it too detailed to describe here.